src/ResourceManagement/ResourceManager/ResourceGroup/ResourceGroupsImpl.cs

```diff
@@ -51,8 +51,7 @@ public async Task<bool> ContainAsync(string name, CancellationToken cancellation
 
         public ResourceGroup.Definition.IBlank Define(string name)
         {
-            ResourceGroupInner inner = new ResourceGroupInner();
-            inner.Name = name;
+            ResourceGroupInner inner = new ResourceGroupInner(default, default, name);
```

Comment:
maybe better to use `ResourceGroupInner(name: name)`, in case there is future change on placing of defaults.

Reply:
We will still have to use **ResourceGroupInner inner = new ResourceGroupInner(location: null, id: null, name: name)**, we can't set name only here.